eclass/bitcoincore.eclass

```diff
@@ -159,7 +162,7 @@ bitcoincore_common_depend_use() {
 	in_bcc_iuse "$1" || return
 	BITCOINCORE_COMMON_DEPEND="${BITCOINCORE_COMMON_DEPEND} $1? ( $2 )"
 }
-bitcoincore_common_depend_use upnp net-libs/miniupnpc
+bitcoincore_common_depend_use upnp '>net-libs/miniupnpc-1.9.20150915'
```

Comment:
> "Breaking" the depgraph here gives the user the option between knowingly keywording a safe version of miniupnpc

We are not breaking the depgraph. Period.



> For reference, here is the Gentoo bug for the security issue. It has mostly been neglected.

That's a reason to call maintainer timeout, not break the depgraph. Since the maintainer is mgorny, we can instead ping him about it, because he is active on github.